makefile.shade

```diff
@@ -98,7 +110,15 @@ var buildTarget = "compile"
                     blockLogger.StartBlock(blockName);
                 }
 
-                Log.Info(blockName);
+                if (IsLinux)
+                {
+                    Exec("copy", "-ar .build " + Path.Combine(repo, ".build"), "");
```

Comment:
`No command 'copy' found, did you mean: ...` where none of the suggestions is the correct `cp`.